lib/mongo/client.rb

```diff
@@ -446,10 +446,18 @@ def hash
     #       JSON Schemas obtained from the server. It protects against a
     #       malicious server advertising a false JSON Schema, which could trick
     #       the client into sending unencrypted data that should be encrypted.
+    #     - Note: If a collection is present on both the encryptedFieldsMap
+    #       and schemaMap, an error will be raised.
     #   - :bypass_auto_encryption => Boolean, when true, disables auto encryption;
     #     defaults to false.
     #   - :extra_options => Hash | nil, options related to spawning mongocryptd
     #     (this part of the API is subject to change).
+    #   - :encrypted_fields_map => Hash | nil, maps a collection namespace to
+    #     an encryptedFields.
```

Comment:
encryptedFields is plural and cannot go with a/an, in this case no article would be used.

However as a user unfamiliar with the feature I don't understand what this is saying, is this a copy-paste from somewhere? Can this be linked to server/etc. documentation?